Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -124,12 +124,26 @@ public enum RenderBlockContent: Equatable {
         public var code: [String]
         /// Additional metadata for this code block.
         public var metadata: RenderContentMetadata?
+        public var copyToClipboard: Bool = true
+
+        public enum OptionName: String, CaseIterable {
+            case nocopy
+
+            init?<S: StringProtocol>(caseInsensitive raw: S) {
```

Comment:
FYI: this can be simplified slightly as
```suggestion
            init?(caseInsensitive raw: some StringProtocol) {
```